Eugene-CG/popup/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="log">
+                Logo
+            </div>
+            <nav class="menu">
+                <ul class="list">
+                    <li class="list__item"><a href="#">Mail</a></li>
+                    <li class="list__item"><a href="#">Images</a></li>
+                    <li class="list__item popup">
+                        <label for="popup-btn__label">
+                            <img src="./img/popup-button.png" alt="">
+                        </label>
+                        <input type="checkbox" class="popup-btn" id="popup-btn">
+                        <ul class="popup-list">
+                            <li class="popup-list__item"><a href="#" class="popup-list__link"><img src="./img/blogs.png" alt="blogs"></a></li>
```

Comment:
Please, delete all alt attrtibutes in popUP and nav icons (img tags). If image is used for decorations and don't have any sanse or information for user, alt attribute should be ampty, without any symbols or spacing.

Reply:
Done